docs/api/websocket_subscribe_api.md

```diff
@@ -1,36 +1,57 @@
 # Lisk Service Subscribe API Documentation
 
-The Lisk Service is a web application that interacts with the whole Lisk ecosystem in various aspects: by accessing blockchain data, storing users' private data, retrieving and storing market data and interacting with social media.
+The Lisk Service is a web application that interacts with the whole Lisk ecosystem in various aspects. Update about blockchain events is one of them.
 
 The Subscribe API is sometimes called publish/subscribe or Event-Driven API. The biggest difference between Event-Driven and regular REST API is not only technical: in practice, they use two-way streaming connection, which means that not only client can request server for a data (and potential updates), but also the server can notify the client about new data instantly, as they arrive.
 
 Lisk Service leverages the two-way communication approach by utilizing the WebSocket library responsible for updating users about changes in the blockchain network and markets.
 
-## Table of Contents
-
-TBD
-
-## Access paths and compatibility
+## <a name='table-of-contents'></a>Table of Contents
+
+<!-- vscode-markdown-toc -->
+* [Table of Contents](#table-of-contents)
+* [Access paths and compatibility](#access-paths-and-compatibility)
+* [Endpoint Logic](#endpoint-logic)
+* [Responses](#responses)
+* [Date Format](#date-format)
+* [Example of a client implementation](#example-of-a-client-implementation)
+    * [Node.js](#node.js)
+* [`update.block`](#`update.block`)
+    * [Response](#response)
+* [`update.round`](#`update.round`)
+    * [Response](#response-1)
+* [`update.transactions.confirmed`](#`update.transactions.confirmed`)
+    * [Response](#response-2)
+* [`update.transactions.unconfirmed`](#`update.transactions.unconfirmed`)
+    * [Response](#response-3)
```

Comment:
I recommend grouping the events into a single section: `Emitted blockchain updates` and make the TOC a little less overwhelming:

```suggestion
* [Emitted blockchain updates](#blockchain-updates-\(`/blockchain`\))
    * [`update.block`](#`update.block`)
    * [`update.round`](#`update.round`)
    * [`update.transactions.confirmed`](#`update.transactions.confirmed`)
    * [`update.transactions.unconfirmed`](#`update.transactions.unconfirmed`)
```

Reply:
It's auto-generated, so it's rather hard to make it nice every time at the refresh.